Description: Drop unfeasible test
 That code works on the original upstream source, which embeds a slightly
 modified version of pygments, but won't work with the version of pygments in
 debian.
 .
 With Debian pygments:
 .
 $ python
 Python 2.7.9 (default, Dec 11 2014, 08:58:12) 
 [GCC 4.9.2] on linux2
 Type "help", "copyright", "credits" or "license" for more information.
 >>> from pygments import lexers
 >>> lexers.guess_lexer('a')
 Traceback (most recent call last):
   File "<stdin>", line 1, in <module>
   File "/usr/lib/python2.7/dist-packages/pygments/lexers/__init__.py", line 252, in guess_lexer
     raise ClassNotFound('no lexer matching the text found')
 pygments.util.ClassNotFound: no lexer matching the text found
 .
 With upstream embeded pygments:
 .
 $ python
 Python 2.7.9 (default, Dec 11 2014, 08:58:12) 
 [GCC 4.9.2] on linux2
 Type "help", "copyright", "credits" or "license" for more information.
 >>> from pygments import lexers
 >>> lexers.guess_lexer('a')
 <pygments.lexers.QBasicLexer>
Author: Antonio Terceiro
Origin: vendor
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/test_pygments.rb
+++ b/test/test_pygments.rb
@@ -87,11 +87,6 @@ class PygmentsHighlightTest < Test::Unit
     assert_match '<title>test</title>', code
   end
 
-  def test_highlight_works_with_single_character_input
-    code = P.highlight("a")
-    assert_match 'a</span>', code
-  end
-
   def test_highlight_works_with_trailing_newline
     code = P.highlight(RUBY_CODE_TRAILING_NEWLINE)
     assert_match '<span class="c1">#!/usr/bin/ruby</span>', code
